utils/from_mommy_to_bakery.py

```diff
@@ -0,0 +1,101 @@
+"""
+Migrate from model_mommy to model_bakery.
+
+``python from_mommy_to_bakery.py --dry-run``
+
+Please check your dependency files.
+"""
+
+import argparse
+import os
+import re
+
+PACKAGE_NAME = r'\bmodel_mommy\b'
+PACKAGE_NAME_PATTERN = re.compile(PACKAGE_NAME)
+PACKAGE_RECIPES = r'\bmommy_recipes\b'
+PACKAGE_RECIPES_PATTERN = re.compile(PACKAGE_RECIPES)
+PACKAGE_LEGACY_MODULE = r'\bmommy\b'
+PACKAGE_LEGACY_MODULE_PATTERN = re.compile(PACKAGE_LEGACY_MODULE)
+
+legacy_and_new = [
+    {'old': PACKAGE_NAME_PATTERN, 'new': r'model_bakery'},
+    {'old': PACKAGE_RECIPES_PATTERN, 'new': r'baker_recipes'},
+    {'old': PACKAGE_LEGACY_MODULE_PATTERN, 'new': r'baker'},
+]
+exclude = [
+    'node_modules', 'venv', '.git', 'sql', 'docs',
+    'from_mommy_to_bakery.py', 'Pipfile', 'Pipfile.lock'
+]
+
+
+def _find_changes(content, pattern, new_value):
+    new_content, substitutions = re.subn(pattern, new_value, content)
+    return new_content, substitutions > 0
+
+
+def _rename_recipe_file(to_be_renamed, dry_run):
+    if dry_run is True:
+        print('Will be renamed:')
+    for old_recipe_file in to_be_renamed:
+        root = old_recipe_file[:old_recipe_file.rfind('/')]
+        if dry_run is False:
+            os.rename(old_recipe_file, f'{root}/baker_recipes.py')
+        else:
+            print(old_recipe_file)
+
+
+def _replace_legacy_terms(file_path, dry_run):
+    try:
+        content = open(file_path, 'r').read()
+    except UnicodeDecodeError:
+        return
+    changed = []
+    for patterns in legacy_and_new:
+        old, new = patterns['old'], patterns['new']
+        content, has_changed = _find_changes(content, old, new)
+        changed.append(has_changed)
+
+    if any(changed):
+        if dry_run is False:
+            open(file_path, 'w').write(content)
+        else:
+            print(file_path)
+
+
+def check_files(dry_run):
+    excluded_by_gitignore = [
+        folder_or_file.strip()
+        for folder_or_file in open('.gitignore').readlines()
+    ]
+    exclude.extend(excluded_by_gitignore)
+
+    to_be_renamed = []
+    for root, dirs, files in os.walk('.', topdown=True):
+        dirs[:] = [
+            directory
+            for directory in dirs
+            if directory not in exclude
+        ]
+
+        for file_ in files:
+            if file_ in exclude or not file_.endswith('.py'):
```

Comment:
I decided not to add this validation because there are other files like `.cfg` and `.yml` that can benefit from the naming replacement. For the requirements file, for instance, the user would have to just update the version number and that's it.

Reply:
I agree with you, but this is not always true. For example, in a few projects from Labcodes, we use the [pip-tools](https://pypi.org/project/pip-tools/) projecto to manage ourt `requirements.txt` files. 

Another thing that bothered me with the replacements in the requirements files is the different version numbers. For example, after the first time I ran the command withouth the `endswith` condition, my requirements was update from `model_mommy=1.6.1` to `model_bakery=1.6.1`. The problem is that there's no version 1.6.1 for the model_bakery lib. 

In my opinion this is somehow misleading since the user will run into issues even thought we're providing a script to migrate the project. To not leave doubts about that, I've added a explicit message addressing that on 04b3d6bee334661d3e49417fa3931f0b14f81360.